tools/data-api/internal/commands/serve.go

```diff
@@ -59,6 +59,10 @@ func (c ServeCommand) Run(args []string) int {
 		serviceNames = pointer.To(strings.Split(serviceNamesRaw, ","))
 	}
 
+	if logLevel := strings.TrimSpace(os.Getenv("PANDORA_LOG")); logLevel != "" {
```

Comment:
I've not made the change to the Data API yet, but the other tools are now using `LOG_LEVEL` fwiw, probably worth doing that here too:

```suggestion
	if logLevel := strings.TrimSpace(os.Getenv("LOG_LEVEL")); logLevel != "" {
```

(in the other tools this is done in `main.go` - but that's a minor nit we can clear up later if needed)

Reply:
Ah cool, I will align it with the rest of the tooling 👍